### ISSUE TITLE
Running HAND FIM

### ISSUE THREAD

**Issue by Pal12344** [NOAA-OWP/inundation-mapping#933]
C:\Users\Anupal>docker run --rm -it --name FIMtutorial2 -v E:/HAND/inundation-mapping/:/foss_fim -v E:/output/:/outputs -v E:/HAND/12090301/branches/5791000001/inputs fimtutorial/inundation-image:latest
Group ID: 1370800235 | Group Name: fim
root@02bfb1e8570b:/# fim_pipeline.sh -u 12090301 -n FIMRUN

======================= Start of fim_pipeline.sh =========================
---- Started: Mon Jun 19 18:27:53 UTC 2023
Traceback (most recent call last):
  File "/foss_fim/src/check_huc_inputs.py", line 97, in <module>
    check_hucs(**args)
  File "/foss_fim/src/check_huc_inputs.py", line 76, in check_hucs
    __check_for_membership(list_hucs, accepted_hucs)
  File "/foss_fim/src/check_huc_inputs.py", line 68, in __check_for_membership
    raise KeyError(msg)
KeyError: 'HUC 12090301 not found in available inputs. Edit HUC inputs or acquire datasets and try again.'

Can i get some suggestion on this?   

Thank you for your time and helping me with the basic understanding

**Comment by RobHanna-NOAA** [comment]
Over the past few days, we have worked directly with Anupal to get this solved. Working with Anupal shows we need some updates including:  
- in ESIP - keeping the full inputs directory, but also have a second "sample_inputs" folder that has only data to process one or two pre-defined HUCs. This brings the total inputs folder size down from 400 GB to less than 55 GB (or even smaller). 
- Update our README.md and other documentation
- Possibly add notes on how to test inundation in our documentation.

Closing this card, and a new card has been created for the corrections.  Issue [936](https://github.com/NOAA-OWP/inundation-mapping/issues/936).
